alica_tests/include/test_alica.h

```diff
@@ -53,6 +53,7 @@ class AlicaTestFixture : public AlicaTestFixtureBase
     alica::AlicaCreators creators;
 
 protected:
+    bool delayStart{false};
```

Comment:
this should be a virtual function that tests can override. Just like other following functions.